test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeGenerator.java

```diff
@@ -23,48 +23,65 @@
 
 import org.objectweb.asm.*;
 
-import jdk.internal.misc.Unsafe;
-
 import static org.objectweb.asm.Opcodes.*;
 
+import java.io.FileOutputStream;
+import java.io.IOException;
 import java.lang.reflect.Field;
 import java.security.ProtectionDomain;
 import java.util.ArrayList;
 import java.util.List;
 
-public class ValueTypeGenerator {
-	private static Unsafe unsafe;
+public class ValueTypeGenerator extends ClassLoader {
+	private static ValueTypeGenerator generator;
 	
 	/* workaround till the new ASM is released */
 	public static final int DEFAULTVALUE = 203;
 	public static final int WITHFIELD = 204;
 	private static final int ACC_VALUE_TYPE = 0x100;
 	
 	static {
-		try {
-			Field f = Unsafe.class.getDeclaredField("theUnsafe");
-			f.setAccessible(true);
-			unsafe = (Unsafe) f.get(null);
-		} catch (NoSuchFieldException | SecurityException | IllegalArgumentException | IllegalAccessException e) {
-			e.printStackTrace();
-		}	
+		generator = new ValueTypeGenerator();
+	}
+	
+	public Class defineClass(String className, byte[] bytes) throws Throwable {
```

Comment:
This function is not necessary.  Forcing resolution will change the behaviour of tests that expect resolution to be performed lazily.